src/target.rs

```diff
@@ -28,6 +28,7 @@ pub enum Os {
     Illumos,
     Haiku,
     Emscripten,
+    Wasi,
```

Comment:
One concern though, in the future if Python gained support for `wasm32-wasi` for extension modules, it will conflicts with this wasmtime based one.

https://github.com/python/cpython/blob/main/Tools/wasm/README.md#wasi-wasm32-wasi

Reply:
Oh, since this is only for bin, it is not an issue.